tests/test_bolt_win.ps1

```diff
@@ -0,0 +1,10 @@
+#$env:PATH += ";C:\Program Files\sensu\sensu-agent\bin"
```

Comment:
why the commented out code?

Reply:
Was left over while debugging Windows issues. I removed.